pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -336,6 +339,8 @@ func getPhase2Binding(vmi *v1.VirtualMachineInstance, iface *v1.Interface, netwo
 		if mac != nil {
 			vif.MAC = *mac
 		}
+
+		tapOwner := "0"
```

Comment:
I understand it is a preparation PR.
In the future PRs, who would calculate the tap owner (AFAIK, it should have the same owner as libvirt)?

Reply:
You can see the current implementation here: https://github.com/kubevirt/kubevirt/pull/4922/files#diff-030aa8b7ff9bf4a511d1d5641c1dec04205a944137672be3a691f0b1b4068753R322 . We tend to sync on VMI objects so I plan to follow this. 

 I am not quite sure why it is relevant to this PR.

Reply:
Now I see the future use case, I don't understand why we need this preparation PR passing the `tapOwner` all the way through binding creation to `createAndBindTapToBridge`.
Why not putting the relevant code calculating the tap owner directly in `createAndBindTapToBridge`?

Reply:
We don't have the `vmi` struct there. I feel I am missing something here.

Reply:
Both `MasqueradeBindMechanism` and `BridgeBindMechanism` have the vmi struct. So it can be passed to `createAndBindTapToBridge`.

Reply:
Ah, I see it now. Thanks! I will simplify it to your suggestion.